i18n/id.toml

```diff
@@ -0,0 +1,195 @@
+# Terjemahan untuk Bahasa Indonesia
+# https://gohugo.io/content-management/multilingual/#translation-of-strings
+
+# === baseof ==
+[backToTop]
+other = "Kembali ke Atas"
+
+[viewComments]
+other = "Lihat Komentar"
+# === baseof ==
+
+# === Post ===
+[posts]
+other = "Pos"
+# === Post ===
+
+# === Taxonomy ===
+[allSome]
+other = "Semua {{ .Some }}"
+
+[tag]
+other = "Label"
+
+[tags]
+other = "Label"
+
+[category]
+other = "Kategori"
+
+[categories]
+other = "Kategori"
+# === Taxonomy ===
+
+# === Pagination ===
+[more]
+other = "Lainnya"
+# === Pagination ===
+
+# === partials/header.html ===
+[selectLanguage]
+other = "Pilih Bahasa"
+
+[switchTheme]
+other = "Ganti Tema"
+# === partials/header.html ===
+
+# === partials/footer.html ===
+[poweredBySome]
+other = "Ditenagai oleh {{ .Hugo }} | Tema - {{ .Theme }}"
+# === partials/footer.html ===
+
+# === partials/comment.html ===
+[valineLang]
+other = "en"
+
+[valinePlaceholder]
+other = "Komentarmu ..."
+
+[facebookLanguageCode]
+other = "id"
```

Comment:
I'm not sure about this, but based on [Facebook internationalization](https://developers.facebook.com/docs/internationalization/) should be `id_ID`

Reply:
I checked again and you're right.